packages/components/.storybook/preview-head.html

```diff
@@ -0,0 +1,4 @@
+<script>
+  // https://github.com/storybookjs/storybook/issues/15391
+  window.global = window;
```

Comment:
Do we still need this even after #1208?

Reply:
Ah, for `jest-mock` apparently.

Reply:
Yes, see https://github.com/storybookjs/storybook/issues/15391